backend/src/controllers/projects/closeout.js

```diff
@@ -27,47 +24,10 @@ controller.notify = async (request, reply) => {
     };
     // const result = ches.send(message);
     const result = message;
-    output = !result ? controller.noQuery(reply, `Notification could not be sent.`) : result;
-  } catch (err) {
-    output = controller.failedQuery(reply, err, what);
-  }
-  return output;
-};
-
-/**
- * Get a specific item by ID.
- *
```

Comment:
@ShawnTurple I moved this one into the useController module, after removing the role check, since it is generic and reusable.